### PR TITLE
Remove duplicate includes

### PR DESCRIPTION
**Pull request by practicalswift**
Remove duplicate includes.

### CODE REVIEW

**Comment by achow101** [comment]
utACK 26441d565faf434a9e05884c767b211bf256824e

**Comment by fanquake** [comment]
utACK 26441d5

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -21,7 +21,6 @@
 #include "timedata.h"
 #include "util.h"
 #include "utilmoneystr.h"
-#include "wallet/coincontrol.h"
```
Comment (kallewoof) [inline src/wallet/rpcwallet.cpp]: Maybe remove the other one instead, to keep the wallet/ imports together?

**Comment by practicalswift** [comment]
@kallewoof Good point! Fixed! :-)

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/10561/commits/d7723abfe08937a4cf82ef9dff43e8b36271c7dd

Travis bug unrelated.

**Comment by kallewoof** [comment]
utACK d7723ab

**Comment by practicalswift** [comment]
Triggered a new Travis build to get rid of the spurious build error.

**Comment by fanquake** [comment]
re-utACK e53a0fa

**Comment by sipa** [comment]
utACK e53a0fa12aa998a73abcf6aac07c79c8077c02d4

**Pull request merged**